src/test/unit/EigenPodManagerUnit.t.sol

```diff
@@ -581,8 +612,8 @@ contract EigenPodManagerUnitTests_BeaconChainETHBalanceUpdateTests is EigenPodMa
 
     function testFuzz_recordPositiveBalanceUpdate(uint sharesBefore, uint sharesDelta, uint prevRestakedBalanceWei) public {
         // Constrain inputs
-        sharesBefore = bound(sharesBefore, 0, type(uint224).max) * uint(GWEI_TO_WEI);
-        sharesDelta = bound(sharesDelta, 1, type(uint224).max) * uint(GWEI_TO_WEI);
```

Comment:
I think this is your issue - we're no longer submitting share amounts divisible by GWEI. So this test is reverting -- guess that's why we were using u224 here.

Should be a simple enough fix, I think you can revert these lines and it should work as intended!